test/com/linkedin/drelephant/util/InfoExtractorTest.java

```diff
@@ -218,14 +218,14 @@ public boolean isEmpty() {
     InfoExtractor.loadSchedulerInfo(result, data, scheduler);
 
     assertEquals(result.scheduler, "azkaban");
-    assertFalse(StringUtils.isEmpty(result.getJobExecId()));
-    assertFalse(StringUtils.isEmpty(result.getJobDefId()));
-    assertFalse(StringUtils.isEmpty(result.getFlowExecId()));
-    assertFalse(StringUtils.isEmpty(result.getFlowDefId()));
-    assertFalse(StringUtils.isEmpty(result.getJobExecUrl()));
-    assertFalse(StringUtils.isEmpty(result.getJobDefUrl()));
-    assertFalse(StringUtils.isEmpty(result.getFlowExecUrl()));
-    assertFalse(StringUtils.isEmpty(result.getFlowDefUrl()));
+    assertFalse(StringUtils.isEmpty(result.jobExecId));
```

Comment:
Mistake of mine. The getters are not explicitly defined, but I guess there is some macros somewhere that generate the getters. I removed my changes.